resources/sass/components/_agenda.scss

```diff
@@ -7,11 +7,18 @@
     color: #6c757d !important;
   }
 
-  .bcl-description-list {
-    font-size: 0.875rem;
-  }
+  .session-content {
+    & > div:last-child,
```

Comment:
This won't be applied in case session has a title only as the title tag is `h4`

Reply:
Fixed in [f774329](https://github.com/openeuropa/oe_whitelabel/pull/268/commits/f77432933c47302ed806f8a1b066d5c3c8755126)